_posts/2016-10-31-twis-82.md

```diff
@@ -0,0 +1,73 @@
+---
+layout:     post
+title:      These Weeks In Servo 82
+date:       2016-10-24 00:30:00
+summary:    What's up with Servo for the weeks of 17 Oct 2016 - 31 Oct 2016
+categories:
+---
+
+In the [last two weeks](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2016-10-17..2016-10-31+user%3Aservo), we landed 246 PRs in the Servo organization's repositories.
+
+The really big news is the [announcement](https://medium.com/mozilla-tech/a-quantum-leap-for-the-web-a3b7174b3c12#.vj4tu2kpu) of Project Quantum. This project will advance the web engine in Firefox, and some of the core pieces of this effort come from Servo! Please read our [mailing list post](https://groups.google.com/forum/#!topic/mozilla.dev.servo/3bH1KaqJq0s) for more information. Or check out the articles from [CNet](https://www.cnet.com/news/firefox-quantum-mozilla-faster-web-gecko-engine/), [Tom's Hardware](http://www.tomshardware.com/news/firefox-quantum-leap-performance-security,32938.html), and others!
+
+Mortimer Goro [finished adding](https://github.com/servo/servo/pull/13840) WebGL support on Windows! He and his team [wrote up](https://blog.mozvr.com/webvr-coming-to-servo-part-1/) a fantastic blog post detailing the architecture, some implementation details, and the overall status.
```

Comment:
Carry on: this blog post is about WebGL on Windows. It's just part of a series on WebVR...